### ISSUE TITLE
System Map

### ISSUE THREAD

**Issue by wolfj95** [the-isf-academy/quest#50]
**Describe what you'd like a resource to explain**
Create a system map to show each thing in the game and the relationships between the things. Also, note where the framework hands off the Arcade framework

**Describe what you want to do with the information**
Useful in directing students to the appropriate place to solve an bug. Diagnosis by tracing the system.

**Describe the requested format**
A high res image on the website or in the docs files

**Anything else?**
Add any other context here.

**Comment by cproctor** [comment]
Good idea! This is a slightly different kind of mapping, but it could be interesting to consider instrumenting the framework with several different levels of logging so students could trace code execution flow.
